### PR TITLE
[MISC] Split hpp/cpp

### PR DESCRIPTION
**Pull request by eseiler**

### CODE REVIEW

File: src/util/display_layout/sizes.cpp [hunk 1]
```diff
@@ -298,7 +308,15 @@ void execute_general_stats(config const & cfg)
     if (!layout_file.good() || !layout_file.is_open())
         throw std::logic_error{"Could not open file " + cfg.input.string() + " for reading"};
 
+// https://godbolt.org/z/PeKnxzjn1
+#if defined(__clang__)
+    auto tuple = chopper::layout::read_layout_file(layout_file);
+    auto filenames = std::get<0>(tuple);
```
Comment (eseiler) [inline src/util/display_layout/sizes.cpp:314]: Should move, same in other cpp

Comment (smehringer) [reply src/util/display_layout/sizes.cpp:314]: This moves, looks like a copy? Or is moving it a todo?

**Review by smehringer — COMMENTED** [review]
In general, you have now added a hpp for every cpp right? I guess we could spare some of the hpps because we don't need to expose most stuff do we?

File: src/util/display_layout/sizes.cpp (hunk 1 above)
Comment (eseiler) [reply src/util/display_layout/sizes.cpp:314]: A todo :)

**Comment by eseiler** [comment]
> In general, you have now added a hpp for every cpp right? I guess we could spare some of the hpps because we don't need to expose most stuff do we?

It could be a follow up. Since this is an application and not a library per se, it's not too important. But "exported" headers could be in include, "internal" headers in src.

**Pull request merged**